server/repository/src/migrations/v2_03_00/remove_stops_from_report_ids.rs

```diff
@@ -0,0 +1,23 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "remove_stops_from_report_ids"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                SELECT
+                    REPLACE(id, '.', '_')
+                FROM 
+                    report;
```

Comment:
```suggestion
                UPDATE report SET id =  REPLACE(id, '.', '_');
                UPDATE form_schema id REPLACE(id, '.', '_'); 
```
Doing just a select does nothing sadly, need to an update!
I think we need to do form_schema as well?

Reply:
Ah nice thank you James. I must have tested this wrong to not pick that up!